src/accelerate/components/Dashboard.scss

```diff
@@ -228,44 +230,179 @@ body.index-php {
 
 .List {
 	position: relative;
-	font-size: 18px;
+	font-size: .875rem;
 	padding: 0 $sidebar-padding 40px;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
```

Comment:
Are these necessary these days? I thought they could be detrimental to readability, at least when not using a web font. At any rate feels like something that would be handled more globally by WP itself.

Reply:
Dropping it to keep consistency with WP styles.

```suggestion
```